waku/v2/node/wakunode2.nim

```diff
@@ -120,7 +120,9 @@ proc init*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   info "Initializing networking", hostAddress,
                                   announcedAddresses
   # XXX: Add this when we create node or start it?
-  peerInfo.addrs.add(hostAddress)
+  peerInfo.addrs.add(hostAddress) # Index 0
+  for multiaddr in announcedAddresses:
```

Comment:
Just curious to know where does the content of `announcedAddresses` change? (except its initialization in [line 117](https://github.com/status-im/nim-waku/blob/8b0b515297a9a1871232b04b9d868e5ccc642d82/waku/v2/node/wakunode2.nim#L117))

Reply:
Yeah, it never does change. :) In fact, up to now it was never used after initialisation, similar to the [eth2 implementation](https://github.com/status-im/nimbus-eth2/blob/7d97228208db770ae5f8b37fb5f47f9f75bb5ee4/beacon_chain/eth2_network.nim#L1541-L1551) that this is based on.